scripts/install-multi-user.sh

```diff
@@ -313,10 +337,15 @@ __sudo() {
 _sudo() {
     local expl="$1"
     shift
-    if ! headless; then
+    if ! headless || is_root; then
         __sudo "$expl" "$*" >&2
     fi
-    sudo "$@"
+
+    if is_root; then
+        env "$@"
```

Comment:
Note this uses `env` because some _sudo callsites pass environment variables to the program being executed.